src/messaging/ReliableMessageContext.cpp

```diff
@@ -265,9 +265,9 @@ CHIP_ERROR ReliableMessageContext::HandleNeedsAckInner(uint32_t messageCounter,
 
         // Replace the Pending ack message counter.
         SetPendingPeerAckMessageCounter(messageCounter);
-        mNextAckTimeTick =
-            static_cast<uint16_t>(CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK +
-                                  GetReliableMessageMgr()->GetTickCounterFromTimeDelta(System::Clock::GetMonotonicMilliseconds()));
+        mNextAckTimeTick = static_cast<uint16_t>(
+            CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK +
+            GetReliableMessageMgr()->GetTickCounterFromTimeDelta(System::SystemClock().GetMonotonicMilliseconds()));
```

Comment:
This uint16_t seems to also be problematic. How is it used?

Reply:
Tagging for issue #10062